### PR TITLE
Add sentinel version handler

### PR DESCRIPTION
**Pull request by jalons**
Add a version handler to sentinel agents to ask them the current running version.

### CODE REVIEW

File: server/zoom/agent/util/helpers.py [hunk 1]
```diff
@@ -99,6 +99,23 @@ def get_log(count=100):
         # return last `count` rows
         return [l.rstrip('\n') for l in lines[-count:]]
 
+
+def get_version():
+    """
+    Return the version of sentinel running
+    :return: str
+    """
+    vfile = os.path.join(os.path.split(os.getcwd())[0:-1][0], 'version.txt')
```
Comment (jalons) [inline server/zoom/agent/util/helpers.py]: Not actually sure this is going to work on windows, but "should"

File: server/zoom/common/handlers.py [hunk 2]
```diff
@@ -39,3 +39,14 @@ def get(self):
...
+class VersionHandler(RequestHandler):
+    def get(self):
+        """
+        @api {get} /version Return the version of the software
+        @apiDescription Return the running version of the software
+        @apiVersion 1.0.0
+        @apiName Version
+        @apiGroup Common
+        """
+        self.write(self.application.version)
```
Comment (jgminer) [inline server/zoom/common/handlers.py]: This is how you get to the tornado.web.Application member variable `version`?

Comment (jalons) [inline server/zoom/common/handlers.py]: Scary right? And non-intuitive? Thank @nicmoski for this one.

File: server/zoom/agent/util/helpers.py (hunk 1 above)
Comment (nicmoski) [inline server/zoom/agent/util/helpers.py]: `os.path.join(os.path.dirname(os.getcwd()), 'version.txt')`?
Slightly more readable, IMO

**Pull request merged**